tests/aws/services/s3/test_s3.py

```diff
@@ -7544,6 +7543,8 @@ def test_presigned_url_v4_x_amz_in_qs(
         # assert that the Javascript SDK hoists it in the URL, unlike Boto
         assert StorageClass.STANDARD in presigned_url
         assert "bar-complicated-no-random" in presigned_url
+        # the JS SDK also adds a default checksum now even for pre-signed URLs
```

Comment:
Default checksum wow

Reply:
Yeah I still don't understand how this made it into prod 😄